testimony.yaml

```diff
@@ -58,6 +58,7 @@ CaseComponent:
     - Infobloxintegration
     - Infrastructure
     - InsightsInventoryPlugin
```

Comment:
'InsightsInventoryPlugin' => this is required because of Polarian?

Reply:
Both are required, as both are valid values - these are two different plugins, both supported.

We don't use `InsightsInventoryPlugin` anywhere now, as this is newer thing. Presumably I will be adding these tests one day in the future.